src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -10,66 +8,109 @@
 import java.util.Collections;
 import java.util.List;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 class AuthorPostSorterTest {
+    UserPost userPost1 = new UserPost("Joe Bloggs",
+            OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+            "Hello World!", 2);
+
+    UserPost userPost2 = new UserPost("Joe Bloggs",
```

Comment:
(Minor) Nice catch to have the same value to check for stability. My only concern with this is that stability is an edge case and so should probably be given a separate couple of tests that explicitly state that they are testing the stability. This is because tests often want to be as clear as possible from the naming and the code as to the reason that they failed, if your standard sorting 3 desc tests failed but it was a stability issue then it might take a lot longer to work out what the issue was as you might think it was desc in general rather than stability